src/libraries/Microsoft.NETCore.Platforms/src/runtimeGroups.props

```diff
@@ -157,21 +147,31 @@
     <RuntimeGroup Include="ol">
       <Parent>rhel</Parent>
       <Architectures>x64</Architectures>
-      <Versions>7;7.0;7.1;7.2;7.3;7.4;7.5;7.6</Versions>
+      <Versions>7;8</Versions>
+      <TreatVersionsAsCompatible>false</TreatVersionsAsCompatible>
       <ApplyVersionsToParent>true</ApplyVersionsToParent>
     </RuntimeGroup>
-    <RuntimeGroup Include="ol">
-      <Parent>rhel</Parent>
+    <!-- The host doesn't strip minors for Oracle Linux.
+         We map them here to the major version or Oracle Linux and RHEL. -->
+    <RuntimeGroup Include="ol.7">
+      <Parent>rhel.7</Parent>
       <Architectures>x64</Architectures>
-      <Versions>8;8.0</Versions>
-      <ApplyVersionsToParent>true</ApplyVersionsToParent>
+      <Versions>0;1;2;3;4;6;6</Versions>
+      <ApplyVersionsToParent>false</ApplyVersionsToParent>
+      <OmitRIDDefinitions>ol;ol.7;ol.7-x64</OmitRIDDefinitions>
+    </RuntimeGroup>
+    <RuntimeGroup Include="ol.8">
+      <Parent>rhel.8</Parent>
+      <Architectures>x64</Architectures>
+      <Versions>0</Versions>
+      <ApplyVersionsToParent>false</ApplyVersionsToParent>
+      <OmitRIDDefinitions>ol;ol.8;ol.8-x64</OmitRIDDefinitions>
     </RuntimeGroup>
 
     <!-- all opensuse releases are independent -->
     <RuntimeGroup Include="opensuse">
       <Parent>linux</Parent>
       <Architectures>x64</Architectures>
-      <Versions>13.2;15.0;15.1;42.1;42.2;42.3</Versions>
       <TreatVersionsAsCompatible>false</TreatVersionsAsCompatible>
```

Comment:
openSuse 15.3 is EOL 2022-12-31, openSuse 42.3 is EOL 2019-07-01 (https://en.opensuse.org/Lifetime#Discontinued_distributions)